### DISCUSSION TITLE
[2021.08.27] 동명 동물 수 찾기

### DISCUSSION THREAD

**Discussion by pyunji** [pyunji/Hyundai-ITnE-Team_QuadCore#71] [SQL]
[동명 동물 수 찾기](https://programmers.co.kr/learn/courses/30/lessons/59041)

**Comment by pyunji** [comment]
[동명 동물 수 찾기 solved](https://github.com/pyunji/Hyundai-ITnE-Team_QuadCore/blob/pyunji/pyunji/sql/PR_59041.sql)

**Comment by leechang4** [comment]
[동명 동물 수 찾기 solved](https://github.com/pyunji/Hyundai-ITnE-Team_QuadCore/blob/master/lch/SQL/PR%2059041.sql)

**Comment by chaeyeon4u** [comment]
[동명 동물 수 찾기 solved](https://github.com/pyunji/Hyundai-ITnE-Team_QuadCore/blob/master/ycy/homework/SQL/PR_59041.sql)

**Comment by hyk339** [comment]
[동명 동물 수 찾기 solved](https://github.com/hyk339/Hyundai-ITnE-Team_QuadCore/commit/487f054701db6e356c81368f1d4469d02587eed6)
